frontend/packages/shared/src/components/AltinnSpinner.tsx

```diff
@@ -7,6 +7,7 @@ import { useTranslation } from 'react-i18next';
 export type IAltinnSpinnerComponentProvidedProps = {
   spinnerText?: string;
   className?: string;
+  altinnSpinner?: boolean;
```

Comment:
This prop doesn't look like it does anything here, can it be removed again?

Reply:
Removed